opm/simulators/wells/MultisegmentWellGeneric.cpp

```diff
@@ -57,6 +57,16 @@ scaleSegmentRatesWithWellRates(const std::vector<std::vector<int>>& segment_inle
     auto& ws = well_state.well(baseif_.indexOfWell());
     auto& segments = ws.segments;
     auto& segment_rates = segments.rates;
+    Scalar sumTw = 0;
+    if(std::any_of(segment_rates.begin(), segment_rates.end(), [](const auto& unscaled_top_seg_rate) {return std::abs(unscaled_top_seg_rate) > 1e-12;})) {
```

Comment:
space before brace, and here have some newlines, they are free.

Reply:
@akva2 where do you want the newlines here? :)

Reply:
```
std::any_of(segment_rates.begin(), segment_rates.end(),
                   [](const auto& unscaled_top_seg_rate) 
                   {
                       return std::abs(unscaled_top_seg_rate) > 1e-12; 
                   });
```
i gave up formatting the code properly but linebreaks are there.